src/common/tvgMath.h

```diff
@@ -44,6 +44,18 @@ bool mathIdentity(const Matrix* m);
 void mathMultiply(Point* pt, const Matrix* transform);
 
 
+static inline float mathDeg2Rad(float degree)
+{
+     return degree * MATH_PI / 180.0f;
```

Comment:
how about re-define `MATH_PI / 180.0f` ?
```
#define MATH_PI3 0.01745329238474369049f
```

Reply:
I don't see any beneficial points. Please think about these points:
1. better performance?
2. less & neat code?
3. enhance the readibility?

Reply:
1. A division operation is performed every time. The amount of computation can be reduced.
2.3.4. You have already defined `MATH_PI *0.5f` as `MATH_PI2`. If you use an appropriate name, I think it is better.

Reply:
- Compiler will do pre-computation. No benefits practically.
- MATH_PI2 has followed the M_PI2, as in the math definition. This prevents the repetitive code from the usage.
- Current logic is already commonized and clear to read. Please suggest me the proper name if you'd like to argue this. 
- In the end, those definition introduction will just increase the LOC.